dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -2193,4 +2284,62 @@ public File generateJson(File targetJsonDir, List<File> esas) throws IOException
         return result;
     }
 
+    private List<File> downloadPublicKeys() {
+        VerifySignatureUtility verifyUtility = new VerifySignatureUtility();
+        VerifyOption verify = (VerifyOption) data.get(InstallConstants.VERIFY_OPTION);
+        Collection<Map<String, String>> usrkeys = (Collection<Map<String, String>>) data.get(InstallConstants.USER_PUBLIC_KEYS);
+        try {
+            pubKeys = verifyUtility.downloadPublicKeys(usrkeys, verify, envMap);
+        } catch (InstallException e) {
+            data.put(InstallConstants.ACTION_RESULT, ERROR);
```

Comment:
if we catch the exception and don't rethrow, how would someone know to check `data` if one of the multiple keys were downloaded?